File: v2/java/src/main/java/vestorly/client/model/MemberEvents.java
package vestorly.client.model;

import vestorly.client.model.MemberEvent;
import vestorly.client.model.TruncatedMember;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class MemberEvents  {
  
  private List<TruncatedMember> members = new ArrayList<TruncatedMember>() ;
  private List<MemberEvent> memberEvents = new ArrayList<MemberEvent>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("members")
  public List<TruncatedMember> getMembers() {
    return members;
  }
  public void setMembers(List<TruncatedMember> members) {
    this.members = members;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("member_events")
  public List<MemberEvent> getMemberEvents() {
    return memberEvents;
  }
  public void setMemberEvents(List<MemberEvent> memberEvents) {
    this.memberEvents = memberEvents;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class MemberEvents {\n");
    
    sb.append("  members: ").append(members).append("\n");
    sb.append("  memberEvents: ").append(memberEvents).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/vestorly/Model/MemberEvents.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class MemberEvents {
    
    
    [DataMember(Name="members", EmitDefaultValue=false)]
    public List<TruncatedMember> members { get; set; }

    
    
    [DataMember(Name="member_events", EmitDefaultValue=false)]
    public List<MemberEvent> memberEvents { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MemberEvents {\n");
      
      sb.Append("  members: ").Append(members).Append("\n");
      
      sb.Append("  memberEvents: ").Append(memberEvents).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}
